es/refresher-linear-algebra.md

```diff
@@ -228,44 +228,43 @@
 
 **39. Linearly dependence ― A set of vectors is said to be linearly dependent if one of the vectors in the set can be defined as a linear combination of the others.**
 
-&#10230; Dependencia Lineal ― Un conjunto de vectores se dice que es linealmente dependiente si uno de los vectores en el grupo puede ser definido como una combinación lineal de los otros.
+&#10230; Dependencia lineal ― Un conjunto de vectores se dice que es linealmente dependiente si uno de los vectores del conjunto puede ser definido como una combinación lineal de los restantes.
 
 <br>
 
 **40. Remark: if no vector can be written this way, then the vectors are said to be linearly independent**
 
-&#10230; Nota: si no se puede escribir el vector de esta manera, entonces el vector se dice que es linealmente independiente
+&#10230; Observación: si ningún vector puede ser escrito de esta forma, entonces se dice que los vectores son linealmente independientes
 
 <br>
 
 **41. Matrix rank ― The rank of a given matrix A is noted rank(A) and is the dimension of the vector space generated by its columns. This is equivalent to the maximum number of linearly independent columns of A.**
 
-&#10230; Rango matricial ― El rango de una matriz A, nombrado rank(A), que es la dimensión del vector espacial generado por sus columnas. Lo que es equivalente al número máximo de columnas linealmente independientes de A.
+&#10230; Rango matricial ― El rango de una matriz A, denotado rank(A), es la dimensión del espacio vectorial generado por sus columnas. Esto es equivalente al máximo número de columnas linealmente independientes de A.
```

Comment:
- "...matriz dada A es denotado rank(A) y es ..."

Reply:
I think this way, literally from English, it's not clear.
Let's wait for a third opinion.

Reply:
I think that Gustavo is saying that the following translation is more faithful to the original text:
> The rank of a given matrix A is noted rank(A) and is the dimension [...]
> El rango de una matriz dad A es denotado rank(A) y es la dimensión [...]
As you can see in the original text, the word "given" is used, there is no commas in "is noted rank(A)" and it says "is noted" (notice the "is").

Reply:
All right, that's a good point for a tie-breaker.